sdk/core/azure-core/src/main/java/com/azure/core/util/CoreUtils.java

```diff
@@ -183,4 +184,17 @@ public static boolean isNullOrEmpty(CharSequence charSequence) {
         }
         return Collections.emptyMap();
     }
+
+    /**
+     * Convenience method to update the input context to disable copying of HTTP response byte buffers. If the input
+     * context is null, a new instance of context is created with a single entry to disable copying of HTTP response
+     * byte buffers.
```

Comment:
Can you add some sort of comment here that mentions this method should be called when building any sync clients because the underlying issue is not a problem in those cases? I think it'll help document why we default to using this in a bunch of our builders.

Reply:
Our plan has just evolved to also do buffer copy on sync clients too. Srikanta is just beginning the process of removing these changes so that sync and async both copy the buffer. This is to remove any possibility of data loss in sync clients when there are intermediate operations during which time the buffer may still be lost.